WordPress/gutenberg
# Add Post Comment block

## Description
Related: https://github.com/WordPress/gutenberg/issues/24101
Laying the foundation for a block-based post comment building. Adds a **Post Comment** block which requires a Comment ID input for now. The **Post Comment** block provides a context for the inner blocks.

As an example, also implemented a very simple **Post Comment Content** block, which uses the context provided by the **Post Comment** block.

## How has this been tested?
1. Open site editor
2. Add **Post Comment** block
3. Input an existing comment's ID
4. Insert **Post Comment Content** block into the **Post Comment** block
5. You should see the content of the comment

## Types of changes
New feature

## Checklist:
